### PR TITLE
[FIX] Fix relation onchange on ir.model

### PR DESCRIPTION
**Pull request by tgroutars**
See https://www.odoo.com/my/task/1837401

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/addons/base/ir/ir_model.py [hunk 1]
```diff
@@ -465,6 +465,13 @@ def _custom_many2many_names(self, model_name, comodel_name):
     def _onchange_ttype(self):
         self.copy = (self.ttype != 'one2many')
         if self.ttype == 'many2many' and self.model_id and self.relation:
+            if not self.env.get(self.relation):
+                return {
+                    'warning': {
+                        'title': _('Model %s does not exist' % self.relation),
```
Comment (mart-e) [inline odoo/addons/base/ir/ir_model.py]: do not put the variable inside the translation clause
https://www.odoo.com/documentation/11.0/reference/translations.html#variables
(also could be super cool if you could reflect the changes in the .pot file too)

Comment (tgroutars) [reply odoo/addons/base/ir/ir_model.py]: Good catch! I updated the whole pot file for the base module, not sure if you usually handle it like that ?

Comment (tgroutars) [reply odoo/addons/base/ir/ir_model.py]: haaaa why do I see this just 5 seconds after trying to push two more commits ?

Comment (aab-odoo) [reply odoo/addons/base/ir/ir_model.py]: @tgroutars no worries, i handle this. I'll do another commit on top of the one i already merged.

**Comment by tgroutars** [comment]
Thanks !

**Pull request closed without merging**